backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Norm.java

```diff
@@ -184,4 +187,27 @@ public static List<RomanNumeral> getReverseSortedValues() {
           .toList();
     }
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    Norm norm = (Norm) o;
+    return document.isEqualNode(norm.document);
+  }
+
+  @Override
+  public int hashCode() {
```

Comment:
nothing really uses the hashCode but as we have overwritten the equals method need to fullfill the contract between the two methods. Document does not implement hashCode so it is comparing the object identities so we needed our own implementation. Just parsing it as html seemed like the best way to me, even if this means that we are now using some packages for this that we don't really want to have in the Domain layer (I added an exception for them for this specific class to the architecture test). 
If someone has a better idea how to do this I'm really interested :) (we want to be able to compare Norms using equals and that norms with the same xml are the same, this massively simplifies writing tests for them and probably also some other things in the future)